packages/xo-server/src/xo-mixins/xen-servers.js

```diff
@@ -389,6 +389,24 @@ export default class {
 
       this.updateXenServer(id, { error: null })::ignoreErrors()
 
+      xapi.once('eventFetchingError', function eventFetchingErrorListener() {
+        const timeout = setTimeout(() => {
+          xapi.xo.uninstall()
+
+          // switch server status from connected to connecting
```

Comment:
What to we want to do that while the status will already be defined as *disconnected* after this delay `xapiMarkDisconnectedDelay`?

Reply:
According to the implementation, the status will not become `disconnected` after this delay.